tests/TestCIT_KCI.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+import numpy as np
+
+import causallearn.utils.cit as cit
+
+
+class TestCIT_KCI(unittest.TestCase):
+    def test_Gaussian(self):
+        X = np.random.randn(300, 1)
+        X1 = np.random.randn(300, 1)
```

Comment:
Could you provide better naming here? :)

X and X1 doesn't look like they are parellel --- how about name it X0 and X1 or X and X_prime?

Of course, there are two hard things in CS: naming things, cache invalidations and off-by-1 error :)

Reply:
Sure, thanks!